app/pages/Page.js

```diff
@@ -0,0 +1,34 @@
+// @flow
+
+import * as React from 'react';
+import type { TranslatorProps } from 'react-i18next';
+import { translate } from 'react-i18next';
+import { Container, Grid } from 'semantic-ui-react';
+
+import NavigationBar from 'core-components/navigation/NavigationBar';
+
+type ChildrenProps = {
```

Comment:
This is called `PassedProps` in other components. Best keep naming as similar as possible.